### PR TITLE
amp-pixel: remove useless width assignment

### PR DESCRIPTION
**Pull request by jridgewell**
We've already `display: none`'d the element with `toggle(this.element, false)`.

### CODE REVIEW

File: builtins/amp-pixel.js [hunk 1]
```diff
@@ -61,8 +61,6 @@ export function installPixel(win) {
             image.src = src;
             image.width = 1;
```
Comment (cramforce) [inline builtins/amp-pixel.js]: This can also go away.

File: builtins/amp-pixel.js [hunk 2]
```diff
@@ -61,8 +61,6 @@ export function installPixel(win) {
...
             image.height = 1;
-            // Make it take zero space
-            this.element.style.width = 0;
```
Comment (cramforce) [inline builtins/amp-pixel.js]: Also this `this.element.style.width = ''` is likely doing nothing.

Comment (jridgewell) [inline builtins/amp-pixel.js]: And `height`?

File: builtins/amp-pixel.js (hunk 1 above)
Comment (jridgewell) [inline builtins/amp-pixel.js]: Removed.

File: builtins/amp-pixel.js (hunk 2 above)
Comment (cramforce) [inline builtins/amp-pixel.js]: I think :)

Was just giving one example.

**Comment by cramforce** [comment]
LGTM

**Pull request merged**